### PR TITLE
Page orientation 24390 release note

### PR DESCRIPTION
**Pull request by dletorey**
### Description

Added experimental release note for @page page-orientation

### Motivation

Working on [1673987](https://bugzilla.mozilla.org/show_bug.cgi?id=1673987)

### CODE REVIEW

**Comment by mathewhodson** [comment]
This should add the preference name `layout.css.page-orientation.enabled`, and it is only enabled by default in Nightly.

**Comment by dletorey** [comment]
> This should add the preference name `layout.css.page-orientation.enabled`, and it is only enabled by default in Nightly.

This feature flag doe not appear to be working

Tested with this example:
https://developer.mozilla.org/en-US/docs/Web/CSS/@page/page-orientation#result

On FF 112.02

Works kind of in Nightly, [but there are some inconsistencies](https://bugzilla.mozilla.org/show_bug.cgi?id=1673987#c14)

File: files/en-us/mozilla/firefox/experimental_features/index.md [hunk 1]
```diff
@@ -718,6 +718,43 @@ See ([Firefox bug 1461589](https://bugzil.la/1461589)) for more details.
...
+  <tbody>
+    <tr>
+      <th>Nightly</th>
+      <td>111</td>
+      <td>Yes</td>
+    </tr>
+    <tr>
+      <th>Developer Edition</th>
+      <td>111</td>
+      <td>Yes</td>
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/experimental_features/index.md:743]:
```suggestion
      <td>No</td>
```

File: files/en-us/mozilla/firefox/experimental_features/index.md [hunk 2]
```diff
@@ -718,6 +718,43 @@ See ([Firefox bug 1461589](https://bugzil.la/1461589)) for more details.
...
+    </tr>
+    <tr>
+      <th>Beta</th>
+      <td>111</td>
+      <td>Yes</td>
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/experimental_features/index.md:748]:
```suggestion
      <td>No</td>
```

File: files/en-us/mozilla/firefox/experimental_features/index.md [hunk 3]
```diff
@@ -718,6 +718,43 @@ See ([Firefox bug 1461589](https://bugzil.la/1461589)) for more details.
...
+    </tr>
+    <tr>
+      <th>Release</th>
+      <td>111</td>
+      <td>Yes</td>
+    </tr>
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/experimental_features/index.md:754]:
```suggestion
    </tr>
    <tr>
      <th>Preference name</th>
      <td colspan="2"><code>layout.css.page-orientation.enabled</code></td>
    </tr>
```

**Review by bsmth — APPROVED** [review]
Thanks @dletorey 🙌🏻

**Pull request merged**